src/components/map/layers/NetworkLayers.tsx

```diff
@@ -100,7 +101,16 @@ export default class NetworkLayers extends Component<INetworkLayersProps> {
 
         return {
             // Layer name 'solmu' is directly mirrored from Jore through geoserver
-            solmu: (properties: any, zoom: any) => {
+            solmu: (properties: any, zoom: number) => {
+
+                const selectedTransitTypes = toJS(this.props.networkStore!.selectedTransitTypes);
+                if (properties.transittypes) {
+                    const transitTypes = TransitTypeHelper
+                        .convertTransitTypeCodesToTransitTypes(properties.transittypes.split(','));
+                    if (selectedTransitTypes.every(type => !transitTypes.includes(type))) {
+                        return this.getEmptyStyle();
+                    }
```

Comment:
Hmm, would this be more understandable using:
if (!transitTypes.some(type => selectedTransitTypes.includes(type))) {
    return ...
}
Or, which do you think is easier to read?
Algorithmically speaking, my suggestion would be faster, since we don't need to loop through all items. But these arrays are so small, so it wont matter, use the one you find is easier to read.

Reply:
This is a good point. I'll change it to some.